.github/workflows/pull_request_soundness.yml

```diff
@@ -57,6 +57,7 @@ jobs:
         fetch-depth: 0
     - name: Mark the workspace as safe
       # https://github.com/actions/checkout/issues/766
+      if: ${{ env.GITHUB_WORKSPACE != '' }}
```

Comment:
I think we might have to make this `act` specific. They recommend doing this

```
if: ${{ !github.event.act }}
```

Reply:
We can leave this how it is now, I'm happy to run in `act` and not try to scrape the commands to run locally.